content/guides/concepts/dnslink.md

```diff
@@ -17,7 +17,7 @@ A DNSLink address looks like an [IPNS](/guides/concepts/ipns) address, but it us
 /ipns/myexampledomain.org
 ```
 
-Just like normal IPFS addresses, they can include links to other files:
+Just like normal IPFS addresses, they can include links to other files — or other types of resources that IPFS/UnivFS supports, like directories and symlinks:
```

Comment:
IPFS/UnixFS\* (I made that typo in my original comment XD)

Also, when I wrote "IPFS/UnixFS", it was just for clarification, I was not suggesting that you should include that exact wording (because UnixFS is more like an implementation detail) -- "**or other types of resources that IPFS supports, like directories and symlinks**" should suffice.

*Disclaimer: I'm just a curious user, not an 'information architect' or anything and I have only recently started learning about IPFS.*

Reply:
Fixed. Thanks again for all of your diligent efforts!